src/main/java/org/reaktivity/nukleus/http2/internal/Context.java

```diff
@@ -247,7 +259,9 @@ public Counters counters()
     }
 
     public Context conclude(
-        Configuration config)
+        Configuration config,
+        int window,
+        int maximumSlots)
```

Comment:
`conclude(...)` should have only one parameter.
Use a setter for each additional configuration item (setter returns `this`) before calling `conclude(config)`.